src/wiki/forms.py

```diff
@@ -567,7 +568,7 @@ class SearchForm(forms.Form):
         required=False)
 
 
-class UserCreationForm(UserCreationForm):
+class UserCreationForm(admin.site._registry[User].add_form if User in admin.site._registry else UserCreationForm):
```

Comment:
Is `admin.site._registry` supposed to be accessed externally?

And using the form from the admin doesn't seem like anything we should intend for public user registration.